src/properties.rs

```diff
@@ -71,12 +74,46 @@ where
 
         self.try_load_prop(&mapping, &path).await
     }
+
+    async fn interface_props(&self, interface: &str) -> Result<Vec<StoredProp>, Error> {
+        let interfaces = &self.interfaces.read().await;
+        let prop = interfaces
```

Comment:
What do you think about renaming the parameter of interface_props from `interface` to `interface_name`? I would like renaming also the `let prop` to `let interface`, I think `prop` is misleading.

Reply:
I agree on the `interface_name`, but for the prop the type is `PropertyRef` which I think is better represented by prop.